src/notebook_unit1.ipynb

```diff
@@ -122,9 +124,11 @@
    "outputs": [],
    "source": [
     "# constants and global functions\n",
-    "thershv = 1e-5\n",
-    "filenames = \"efield.t\", \"expec.t\", \"npop.t\", \"nstate_i.t\", \"table.dat\"\n",
+
+    "thershv = 1.e-5\n",
+    "fldr = \"/home/olga/sustainable_development_course/course/team13/\""
     "filedir = 'D:/course/team13/data/'"
+     "filenames = \"efield.t\", \"expec.t\", \"npop.t\", \"nstate_i.t\", \"table.dat\"\n",
```

Comment:
ideally we define all paths relative to _src/_, so that it works on all systems. I would do something like `filedir = "../data/"`, because _data_ is a neighbour folder of _src_.